### ISSUE TITLE
Add Filter by TOP X Active users per day / instance

### ISSUE THREAD

**Issue by calr0x** [jheidecker/lemmony#9]
Can you enable some type of limit to only get the top X communities? The app tried to get 7000 communities and I'm just not sure I need THAT many.. :)

**Comment by jheidecker** [comment]
This is what the community data looks like:

```json
{
        "baseurl": "cigar.cx",
        "url": "https://cigar.cx/c/newworldcigars",
        "name": "newworldcigars",
        "title": "New World Cigars",
        "desc": "",
        "icon": "https://cigar.cx/pictrs/image/349210a1-067c-4533-92ca-ede73fb855d0.webp",
        "banner": "https://cigar.cx/pictrs/image/ac9fbc84-63a3-4881-b4b7-530763266805.jpeg",
        "nsfw": false,
        "counts": {
            "id": 2,
            "community_id": 3,
            "subscribers": 6,
            "posts": 0,
            "comments": 0,
            "published": "2023-06-10T00:13:26.727280",
            "users_active_day": 0,
            "users_active_week": 0,
            "users_active_month": 0,
            "users_active_half_year": 0
        },
        "time": 1688557440139,
        "isSuspicious": false,
        "score": 282
    }
```

and kbin:

```json
{
        "actor_id": "https://fedia.io/m/sanantonio",
        "title": "San Antonio",
        "name": "sanantonio",
        "preferred": "sanantonio",
        "baseurl": "fedia.io",
        "summary": "Anything and everything having to do with the city of San Antonio in Texas.",
        "sensitive": false,
        "postingRestrictedToMods": false,
        "icon": null,
        "published": "2023-06-16T04:18:39+02:00",
        "updated": "2023-06-16T04:24:08+02:00",
        "followers": 2,
        "time": 1688556920647
    }
```

Currently the only filtering that happens is for communites where posts = 0. We could filter using any of these values. Thoughts on how to get "top?" Should it be posts, or active users?

**Comment by calr0x** [comment]
I would probably start with active users but maybe a choice wouldn't be too
bad?

Thanks!

**Comment by calr0x** [comment]
So I'm thinking active users is probably going to be the best metric. It's just a couple hundred communities before you start getting to dead ones.

A use case might be to have the script run once a week and scrape the top X active user communities the past week. That way you just tend to get the most active. Or we could actually set a threshold and pull anything above that.

I might take a look at what the website gives in terms of data and see if I can't help submit some functionality for this project..

The bottom line is we certainly don't have to subscribe to everything and can probably get 95% of the active communities within a few hundred results.

Let me know what you think!

**Comment by jheidecker** [comment]
I think it's a good start! Let me see what I can get done for adding simple filtering for top X active users with another parameter!

**Comment by calr0x** [comment]
Thanks, Jay!

**Comment by calr0x** [comment]
Nice!

Thank you!
